.pre-commit-config.yaml

```diff
@@ -5,25 +5,22 @@ repos:
     - id: check-merge-conflict
     - id: trailing-whitespace
 - repo: https://github.com/python/black
-  # refrain from using 19.10b0 or later until
-  #   https://github.com/psf/black/issues/1288
-  # is fixed
-  rev: 19.3b0
+  rev: 20.8b1
```

Comment:
https://github.com/psf/black/issues/1288 was resolved in the 20.8b0 [release](https://github.com/psf/black/issues/1288)

Reply:
It took a long time but I've been happy with the behavior in the `>=20.8b0` versions.
We should check CLI for this as well. I bet it's still on 19.3b0